spec/duplicates_spec.rb

```diff
@@ -1,3 +1,4 @@
+require File.expand_path('../spec_helper', __FILE__)
```

Comment:
Ok nice. I believe `rspec` adds the `spec/` path to the require paths, so a simple `require 'spec_helper'` should work.